docs/app/components/header/Header.vue

```
@@ -10,6 +10,13 @@ function toggleChat() {
   }
   open.value = !open.value
 }
+
+defineShortcuts({
```

ESLint

[error] 'defineShortcuts' is not defined.

(no-undef)